docs/data-sources/groups.md

```diff
@@ -30,16 +30,29 @@ data "azuread_groups" "allGroups" {
 }
 ```
 
+*Look up all security groups*
+```terraform
+data "azuread_groups" "allGroups" {
+  return_all       = true
+  security_enabled = true
+}
+```
+
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `display_names` - (Optional) The display names of the groups.
 * `object_ids` - (Optional) The object IDs of the groups.
 * `return_all` - (Optional) A flag to denote if all groups should be fetched and returned.
+* `security_enabled` - (Optional) A flag to denote if only `security_enabled=true` groups should be returned.
+* `mail_enabled` - (Optional) A flag to denote if only `mail_enabled=true` groups should be returned.
 
 ~> One of `display_names`, `object_ids` or `return_all` should be specified. Either of the first two _may_ be specified as an empty list, in which case no results will be returned.
 
+~> `security_enabled` and `mail_enabled` flags work with `return_all` and `display_names` but not `object_ids`
```

Comment:
This should be an info box (`~>` makes a red box)

```suggestion
-> `security_enabled` and `mail_enabled` flags work with `return_all` and `display_names` but not `object_ids`
```

Reply:
I had no idea that there were info boxes - Apologies